base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java

```diff
@@ -193,13 +197,8 @@ public boolean send(String host, int port, String servlet, String rawData,
                     wizardInfo.put(name, value);
                 }
             }
-            bstream.close();
             String sendStatus = (String)wizardInfo.get("NMC_STATUS");
-            if (sendStatus.equals("0")) {
-                return true;
-            } else {
-                return false;
-            }
+            return sendStatus.equals("0");
         } catch (Exception e) {
```

Comment:
Something should probably get logged here (same with in other method)

Reply:
I have added in a print of the `sendStatus` value before returning it, The class doesn't define a logger, so I didn't introduce one.